File: hash-table/two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        // answer
        for i in nums{
            for j in (nums-1){
                if i+j == target{
                    return [i,j]
                }
            }
        }
        
    }
};

File: hash-table/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {

        int len = nums.length;

        // check all the possible combination
        for (int i = 0; i < len; i++) {
            for (int j = i + 1; j < len; j++) {
                if (nums[i] + nums[j] == target) {
                    return new int[]{i,j};  //return if found
                }
            }
        }
        return new int[]{};
    }
}